.github/workflows/build.yaml

```diff
@@ -44,7 +44,7 @@ env:
   GHCR_ACTOR: rad-ci-bot
 
   # Container registry url for GitHub container registry.
-  CONTAINER_REGISTRY: ${{ github.event_name == 'pull_request' && 'ghcr.io/radius-project/dev' || 'ghcr.io/radius-project' }}
+  CONTAINER_REGISTRY: 'ghcr.io/radius-project'
```

Comment:
We are removing **dev** because PRs will no longer push to the container registry, right?

Reply:
The functional test workflow pushes to the registry for PRs instead. That avoids the permissions problem because that workflow has access to our secrets.

The push being done here was superfluous for the PR case.